### PR TITLE
Update website with new value proposition

### PR DESCRIPTION
**Pull request by choldgraf**
This overhauls our website by reusing as much of the content from [our value proposition slides](https://docs.google.com/presentation/d/1kjxPAS7ZptK1OLPiAWmi7WjS5EaDHQtg3E_Gig8ldzE/edit?usp=sharing) as possible. I intended for this to be a very quick update, but after looking at the slides I thought it might be more effective if we just tried to adapt them to our website content in one go.

This changes a lot, so your best bet is to just look at the PR preview at the link below and give high-level feedback:

[Preview of the changes is here](https://deploy-preview-239--cocky-booth-e7ed17.netlify.app/)

Here's a brief update of major changes:

- Reframe the message around the "network of communities" value proposition
- Shrink number of top-level pages. It is now Home, cloud platform, community network, right to replicate, and "about"/
- Make all of the pages more image-heavy and remove text in general.
- Add demonstration videos for major functionality on our platform page.
- Add case studies from a few major partners (using the slides in our deck as inspiration)
- Add more detail about the communities we serve, and add logos from a few major ones.
- Add a dedicated page for joining the network: will be https://2i2c.org/join
- Added a short blog post about the value proposition

### Guidance for feedback

- Is there anything major that this update gets wrong or is missing?
- Should we explain the "join the network" call-to-action better? Should we include more information about pricing etc?
- Is there any significant change you'd like to make?
- Should the blog post be significantly different / longer / shorter / etc?

In addition I welcome any small changes or edits as well, but the main thing I want to ensure is that the big stuff feels right here.

### CODE REVIEW

File: content/_index.md [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+            <li class="list-inline-item"><a href="https://www.si.edu/" target="_blank"><img src="/images/logos/community/smithsonian.svg" alt="Smithsonian logo"></a></li>
+            <li class="list-inline-item"><a href="https://www.hhmi.org/" target="_blank"><img src="/images/logos/community/howard-hughes-medical-institute.svg" alt="Howard Hughes Medical Institute logo"></a></li>
+            <li class="list-inline-item"><a href="https://ncas.ac.uk/" target="_blank"><img src="/images/logos/community/ncas.png" alt="NCAS logo"></a></li>
+            <li class="list-inline-item"><a href="https://pangeo.io/" target="_blank"><img src="/images/logos/community/pangeo.svg" alt="Pangeo logo"></a></li>
+            <li class="list-inline-item"><a href="https://www.utoronto.ca/" target="_blank"><img src="/images/logos/community/university-of-toronto.svg" alt="University of toronto logo"></a></li>
+            <li class="list-inline-item"><a href="https://www.utoronto.ca/" target="_blank"><img src="/images/logos/community/university-of-toronto.svg" alt="University of toronto logo"></a></li>
```
Comment (colliand) [inline content/_index.md]: Toronto was repeated twice. Let's remove this line.

**Review by colliand — CHANGES_REQUESTED** [review]
Oh golly! This is beautiful. I saw one needed change: The U. Toronto logo was repeated twice. It's a great improvement.

File: content/blog/2024/value-proposition/index.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  </p>
+  <p class="pull-subtitle">
+  Our interactive computing platform gives your community a digital home to create and share knowledge, and a global network of communities to learn from.
+  </p>
+</div>
+
+We believe that this leans into and builds upon the way we've described our service so far.
+It recognizes that the value in our service doesn't end with simple access to an interactive computing environment, but lies in how that environment's tools empower a community to create and share knowledge with one another.
+
+We believe that this is an exciting direction to build towards, because it is based in the value of the collection of communities we serve, and the way that this network can be greater than the sum of its parts.
```
Comment (Gman0909) [inline content/blog/2024/value-proposition/index.md:46]: The use of "We believe" twice in quick proximity hits a little odd to me, I would just start this sentence with "This is an exciting direction..."

File: content/blog/2024/value-proposition/index.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+We're excited to lean into this direction in how we shape our platform, what open source contributions we make, and where we aim to have impact in the coming years.
+
+To begin, we've interwoven some ideas from this value proposition into 2i2c.org, and the way that we describe our service.
+We'd love feedback from anyone who has an idea, a critique, or "yes, and" to share.
+You can also find [a version of the value proposition in google slides](https://docs.google.com/presentation/d/1C-TNSI2h0181y6-z5tL9AWSRsWp9IfZqCDtcIZEs1MM/edit?usp=sharing) made for a recent talk with the [Incentivizing Collaborative and Open Research](https://incentivizingopen.org/) community.
```
Comment (Gman0909) [inline content/blog/2024/value-proposition/index.md]: I would change "the value proposition" to "our value proposition"

**Comment by jnywong** [comment]
> - Is there anything major that this update gets wrong or is missing?

I love the addition of the videos on the "platform" page, very slick demonstration of our value in action! I think the division of content between the homepage and the "platform page" is the right balance. 

> - Should we explain the "join the network" call-to-action better? Should we include more information about pricing etc?

Yes, but I think what we have for now is right. Perhaps @colliand could implement a make.com > AirTable automation here for leads once that is established. I think pricing is definitely a question folks will ask and should be displayed here once alpha product and pricing is finalised.

> - Is there any significant change you'd like to make?

I would make the first slide/image on the homepage fill the width of the container, as the text within it is relatively small. I would also remove the "Learn about our interactive computing platform" button below it since there are the same buttons at the head and foot of the page.

> - Should the blog post be significantly different / longer / shorter / etc?

The blog post is the right length and conveys an aspirational narrative 👍

**Comment by Gman0909** [comment]
I noticed  that in my dark mode browser a lot of the logos render poorly, maybe something worth looking at:

image


I'm also getting a rendering error in the preview image here:

image

File: content/_index.md [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+
+
+  - block: markdown
+    id: numbers
+    content:
+      title: We serve a global network of communities
+      subtitle: 
+      text: |
+
+        We serve **over 80 communities across the globe** with **over 7000 active users** that are dedicated to creating and sharing knowledge. This includes **educational communities** that share knowledge within a community, **research communities** that create advances at the frontier of knowledge, and **applied knowledge communities** that use advances in knowledge to serve the public.
```
Comment (Gman0909) [inline content/_index.md:80]: This is the third section of the page that mentions "communities" but the first to specifically mention research and education communities. Should we maybe make that clear earlier on?

Comment (choldgraf) [reply content/_index.md:80]: I'll add another "research and education" shout-out to the section just above

**Comment by choldgraf** [comment]
Thanks everybody for your comments - I think that I've addressed everything in there, please let me know if there's anything else you'd like to see improved before merging!

https://deploy-preview-239--cocky-booth-e7ed17.netlify.app/

**Comment by Gman0909** [comment]
I'm still seeing empty images:

image

image

Otherwise it's looking good!!

**Comment by Gman0909** [comment]
Looking great! I briefly commented I was seeing empty images on my browser but it was a caching issue on my side. Great work!

**Comment by choldgraf** [comment]
OK, merging away, thanks all for the comments!

**Pull request merged**